core-bundle/tests/Controller/FrontendModule/TwoFactorControllerTest.php

```diff
@@ -179,6 +179,7 @@ public function testRedirectsAfterTwoFactorHasBeenDisabled(): void
         /** @var RedirectResponse $response */
         $response = $controller($request, $module, 'main', null, $page);
 
+        $this->assertSame('', $user->useTwoFactor);
```

Comment:
Maybe we can also assert here that the backup codes are removed?

Reply:
Yep! Thinking about to check as well if the token version has increased.

Reply:
Added in 63ae617b0. Also a check, if `->clearTrustedDevices()` is called.